tracer/src/Datadog.Trace/ClrProfiler/ServerlessInstrumentation/AWS/Lambda.cs

```diff
@@ -0,0 +1,83 @@
+// <copyright file="Lambda.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Net;
+
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.ClrProfiler.ServerlessInstrumentation.AWS
+{
+    /// <summary>
+    /// Lambda customer handler calltarget instrumentation
+    /// </summary>
+    public class Lambda
```

Comment:
After adding the following using statement to the ones above, can you also apply the following attributes? This will make sure that the class is public _but_ it won't surface in code suggestions to consumers of the SDK.

Place above:
```csharp
using System.ComponentModel;
```

```suggestion
    [Browsable(false)]
    [EditorBrowsable(EditorBrowsableState.Never)]
    public class Lambda
```

Reply:
This change should also require you to revert changes to `tracer/test/Datadog.Trace.Tests/Snapshots/PublicApiTests.PublicApiHasNotChanged.verified.txt` for the unit tests to pass

Reply:
✅